### PR TITLE
Fix sidekiq docs for async adapter and reformat

### PR DESCRIPTION
**Pull request by schmijos**
Using the [async adapter](https://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/AsyncAdapter.html) in development seems to be closer to reality than `inline`.

### CODE REVIEW

File: ruby_on_rails/sidekiq.md [hunk 1]
```diff
@@ -6,116 +6,123 @@ We use [sidekiq](https://github.com/mperham/sidekiq) because it works well on He
...
+  ```
 
-* configure the active job adapter in `config/environments/development.rb`
+* Configure the active job adapter in `config/environments/development.rb`
 
-```rb
-config.active_job.queue_adapter = :inline
-```
+  ```rb
+  config.active_job.queue_adapter = :async
```
Comment (schmijos) [inline ruby_on_rails/sidekiq.md:29]: This is the logical change.

**Review by macav — COMMENTED** [review]
~~I think it was `inline` on purpose, as you don't need to be `sidekiq` running to see the async stuff happen in development (for example if the app only sends e-mails). I think that works fine until you have something more complex, such as `sidekiq-cron`.~~

Sorry, I just opened the documentation for that. From that, it seems to be quite close to `inline`, so fine for me.

**Comment by schmijos** [comment]
> ~I think it was `inline` on purpose, as you don't need to be `sidekiq` running to see the async stuff happen in development (for example if the app only sends e-mails). I think that works fine until you have something more complex, such as `sidekiq-cron`.~
> 
> Sorry, I just opened the documentation for that. From that, it seems to be quite close to `inline`, so fine for me.

It uses `concurrent-ruby` to execute the jobs in parallel. `inline` executes them sequentially.

**Review by coorasse — APPROVED** [review]
I don't recall why we decided for `inline` and I don't see why not using `async`

**Review by sihu — APPROVED** [review]

**Pull request merged**